src/read_config_file.c

```diff
@@ -614,9 +614,16 @@ static PetscErrorCode ValidateConfig(MPI_Comm comm, RDyConfig *config) {
 static PetscErrorCode SetAdditionalOptions(RDy rdy) {
 #define VALUE_LEN 128
   PetscFunctionBegin;
-  PetscBool has_param;
+  PetscBool has_param, locRefine;
```

Comment:
We should use `snake-case` for variables. I can handle this and the other instances of `camelCase`.